frontend/src/Components/loginForm.tsx

```diff
@@ -2,21 +2,25 @@ import React, { useState } from 'react';
 import axios from 'axios';
 import './login.css';
 
+
 export const LoginForm = () => {
     const [email, setEmail] = useState<string>('');
     const [password, setPassword] = useState<string>('');
-    
+    const [errorMessage, setErrorMessage] = useState<string>('');
+
     const handleSubmit = async (event: React.FormEvent<HTMLFormElement>) => {
         event.preventDefault();
         try {
           const response = await axios.post("http://localhost:8000/login/password", {
             email,
             password,
           });
-          console.log(response.data)
           // TODO T105 (FK) After successful response redirect user to whatever page they're trying to reach
+          console.log(response.data)
```

Comment:
is this a debug console log? let's rm if it isn't necessary

Reply:
yes will remove.